app/views/pages/crayons.html.erb

```diff
@@ -0,0 +1,172 @@
+<% title "Crayons" %>
```

Comment:
I'm happy that you created this page for illustrate the **Crayons** design system :)

Reply:
It was meant to be more for myself so I can test it somehow :) But I'll definitely keep it there so we can present components to others :)